addons/website/views/snippets/s_image_punchy.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image_punchy" name="Image Punchy">
+    <section class="s_image_punchy o_cc o_cc2 pt64 pb64">
+        <div class="container">
+            <div class="row o_grid_mode" data-row-count="14">
+                <div class="o_grid_item o_grid_item_image g-height-11 g-col-lg-12 col-lg-12" style="grid-area: 1 / 1 / 12 / 13; z-index: 1">
+                    <img src="/web/image/website.s_image_punchy_default_image" class="figure-img img-fluid rounded" alt=""/>
+                </div>
+                <div class="o_grid_item g-height-6 g-col-lg-7 col-lg-7" style="grid-area: 9 / 6 / 14 / 13; z-index: 2;">
```

Comment:
@anso-odoo  Maybe I'm wrong but I think g-height would be 5 and not 6, since 14-9 = 5. To check.